### PR TITLE
Update how-to-detect-win11.md with Critical-CH header

### PR DESCRIPTION
**Pull request by majenny**
First draft for initial review - need to add code sample. 

**Rendered new article section for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/how-to-detect-win11?branch=pr-en-us-1631#optimizing-detection-performance

**Writer/Editor pass for the new section:**
vchapel

### CODE REVIEW

File: microsoft-edge/web-platform/how-to-detect-win11.md
Comment (naarcini) [inline microsoft-edge/web-platform/how-to-detect-win11.md:79]: I can't quite parse this line. Nothing is sent in `Accept-CH` by default. `Accept-CH` is a response header that a server may optionally add. If the browser sees this header in the response, the next request may include a `Sec-CH-UA-Platform-Version` request header. Could you clarify?

Your clarification might also help with the below explanation of `Critical-CH` since I'm similarly confused there.

**Review by naarcini — CHANGES_REQUESTED** [review]

File: microsoft-edge/web-platform/how-to-detect-win11.md
Comment (vchapel) [reply microsoft-edge/web-platform/how-to-detect-win11.md:79]: From Mozilla "The Accept-CH header may be set by a server to specify which client hints headers a client should include in subsequent requests." So it sounds like there are no default fields or values. I will try a short introduction sentence, with some minor wording changes to the rest of this section. Then you and Jenny can let me know if it is an improvement.

Comment (vchapel) [reply microsoft-edge/web-platform/how-to-detect-win11.md]: I think Jenny's edits resolved this.

**Comment by majenny** [comment]
Link to update request svg's: https://bit.ly/32ZNLNZ https://bit.ly/3eNM6xK

**Comment by vchapel** [comment]
@majenny My change caused a branch merge conflict. I think it is just the ms.topic article type ... I believe it should be "article" rather than "concept". I would resolve the conflict but I don't have write access to your branch. Thanks!

**Review by vchapel — APPROVED** [review]
LGTM, pending approvals @majenny @naarcini

**Comment by majenny** [comment]
LGTM!

**Comment by vchapel** [comment]
@majennny do we need re-review from @naarcini or is it ready to merge.

**Review by naarcini — APPROVED** [review]

**Comment by naarcini** [comment]
Sorry for taking so long, LGTM!

**Comment by vchapel** [comment]
No worries, thanks!


[A close up of graphics  Description automatically generated]
Victor Chapel (Experis)
Technical Writer
Microsoft Edge

Alias
💻 v-vchapel
Working hours
***@***.***  11AM - 7PM (Central US)
🌉 9AM - 5PM (Pacific US)
🌍 5PM - 1AM (UTC+1)
Pronouns
🆗 he/him  🆗 they/them

**Pull request merged**
